brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
@arsenius7, Does the stride need to be a timestep or the number to frame to be advanced?

To me using time for the stride is bogus. What sense does it have to use a stride of timestep * 1.1 if there's no frame interpolation? That simply means that the frame skip is a bit unpredictable.